react/src/base/inputs/SprkTextarea/SprkTextarea.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import uniqueId from 'lodash/uniqueId';
+
+const SprkTextarea = ({
+  additionalClasses,
+  analyticsString,
+  children,
+  forwardedRef,
+  idString,
+  isDisabled,
+  id,
+  value,
+  formatter,
+  ariaDescribedBy,
+  isValid,
+  ...rest
+}) => {
+  return (
+    <textarea
+      className={classNames('sprk-b-TextArea', additionalClasses, {
+        'sprk-b-TextArea--error': !isValid,
+      })}
+      id={id}
+      disabled={isDisabled}
+      ref={forwardedRef}
+      data-id={idString}
+      value={isValid && formatter(value) ? formatter(value) : value}
+      data-analytics={analyticsString}
+      aria-invalid={!isValid}
+      aria-describedby={ariaDescribedBy}
+      {...rest}
+    >
+      {children}
```

Comment:
I'm not sure we need to pass `children` through on `textarea`. If I don't pass `value` then React throws an error, and if I do pass `value` then `children` is ignored.

Reply:
I'm going to remove children. In HTML textareas use the children and not value. But in React they use value. I messaged Katie about this discovery yesterday and then forgot to update the component lol. Thanks for catching.